addons/misc/CfgWeapons.hpp

```diff
@@ -56,12 +56,16 @@ class CfgWeapons
         picture = QPATHTOF(ui\ArmbandKat.paa);
     };
 
-    class kat_Bubble_Wrap_Item: kat_Armband_Red_Cross_Item {
+    class kat_Bubble_Wrap_Item: ItemCore {
 
+        detectRange = -1;
```

Comment:
if somehow AI gets hold off this weapons this disables it being able to be used.

Reply:
I see. Then in this case, wouldn't it be better to utilize the ACE_ItemCore same as here (https://github.com/KAT-Advanced-Medical/KAM/blob/4b5f4d27616b3cfff9662090c8c10ab61d41d800/addons/pharma/CfgWeapons.hpp#L5)? This would solve all my three comments, unless there is some kind of issue with the ACE_ItemCore as well (Causing issue with the AI).

Reply:
As far as I am aware of there is none, I changed the according lines.

Reply:
Perfect, thank you very much 👍🏻